internal/testplanet/uplink.go

```diff
@@ -330,6 +330,7 @@ func atLeastOne(value int) int {
 func (client *Uplink) NewLibuplink(ctx context.Context) (*libuplink.Uplink, error) {
 	config := getDefaultConfig()
 	libuplinkCfg := &libuplink.Config{}
+	libuplinkCfg.Volatile.Log = client.Log
```

Comment:
Can we name this log? From the build logs it looks like it is not named and thus printed with the global logger.

Reply:
Hmm, it should be using client.Log, which is named.

Reply:
Let me take a look at the logs again, maybe i mixed something up

Reply:
Oh seems like my fault: https://build.ops.storj.io/blue/organizations/jenkins/storj/detail/PR-2676/1/pipeline/43#step-59-log-839
Thats the output of the actual stdout and stderr of the test using the gateway/uplink